docs/unified_api.md

```diff
@@ -0,0 +1,105 @@
+# Unified API
+
+## Problem
+Currently the Childchain API and Watcher API behave differently e.g.
+ - Childchain API:
+   - Must have jsonrpc and id in the request body
+   - **Cannot** have 'Content-Type': 'application/json' header
+   - On error, response contains response.error
+ - Watcher API:
+   - **Must** have 'Content-Type': 'application/json' header
+   - On error, response contains response.result === 'error'
+
+Also on the roadmap is the **Informational API Service** that will provide non-critical convenience APIs.
+
+All three APIs should behave consistently.
+  
+## Proposal
```

Comment:
Not sure if it's appropriate in this document - an option to be able to set CORS (for which components)?

Reply:
Besides the benefits CORS could bring... Enabling CORS for all paths will be easy and should not affect this design. 

However if we would like to select the paths that are enabled for CORS - this approach has a drawback proposing all calls to be made with POST.  For instance it will be easier to configure enabling CORS just for GET request than to switch it by the path.